src/components/contact/contactFAQ/ContactFAQ.tsx

```diff
@@ -0,0 +1,134 @@
+            </Accordion.Body>
+          </Accordion.Item>
+          <Accordion.Item eventKey="2">
+            <Accordion.Header>
+              <b>How do I block or unblock someone?</b>
+            </Accordion.Header>
+            <Accordion.Body>
+              Adipisicing consectetur labore veniam pariatur. Aliqua ad aute
+              labore consectetur magna. Voluptate ullamco ad ad fugiat in
+              officia velit aliquip Lorem deserunt aute dolore adipisicing. Elit
+              cupidatat dolore aliqua nulla ea nulla quis Lorem Lorem velit.
+              Reprehenderit mollit labore non ad eiusmod proident aute mollit
+              qui officia do laborum id excepteur. Fugiat nostrud dolor proident
+              non veniam et quis minim adipisicing. Deserunt commodo dolore
+              exercitation laborum cupidatat eiusmod irure nostrud.
+            </Accordion.Body>
+          </Accordion.Item>
+          <Accordion.Item eventKey="3">
+            <Accordion.Header>
+              <b>How do I donate something other than money?</b>
+            </Accordion.Header>
+            <Accordion.Body>
+              Ad laboris fugiat do dolore sunt dolor ex est minim occaecat
+              deserunt. Consequat quis dolor velit nulla mollit. Nostrud eiusmod
+              eu sit aute consequat cupidatat ex sint id esse labore tempor
+              nostrud fugiat. Nostrud id fugiat id Lorem deserunt eu fugiat
+              voluptate nulla minim. Exercitation ipsum commodo duis occaecat et
+              eu culpa esse. Irure et dolor anim consequat officia esse eiusmod
+              laboris ad exercitation magna ipsum laborum.
+            </Accordion.Body>
+          </Accordion.Item>
+          <Accordion.Item eventKey="4">
+            <Accordion.Header>
+              <b>I am having trouble adding a resource</b>
+            </Accordion.Header>
+            <Accordion.Body>
+              Cillum laboris mollit velit deserunt dolore dolore. Officia in
+              exercitation excepteur mollit deserunt. Culpa labore veniam eu ad
+              id labore irure id culpa deserunt aute.
+            </Accordion.Body>
+          </Accordion.Item>
+          <Accordion.Item eventKey="5">
+            <Accordion.Header>
+              <b>How do I change my username or email address?</b>
+            </Accordion.Header>
+            <Accordion.Body>
+              <p>To change your username or email:</p>
+              <ol>
+                <li>
+                  Go to your profile by clicking your profile icon in the top
+                  right corner. ( On mobile, this is in the hamburger menu: ☰ )
+                </li>
+                <li>Click account settings.</li>
+                <li>Scroll down and click privacy.</li>
+                <li>Edit your username or email address as desired.</li>
+              </ol>
+              <p>
+                NOTE: updating your username will require confirmation by email.
```

Comment:
Is this logic that we'll need to implement at a later date?

Reply:
I'm not actually sure about the logic, this just came directly from the mockups (I'm not sure of its origin and if it is actually agnostic to previous logic or not)

Reply:
Yeah, we can confirm if that's desired behavior at a future meeting.